main.py

```diff
@@ -1,6 +1,10 @@
 from flask import Flask, render_template
+import webapp2
 from google.appengine.ext import ndb
 from models import Coffee
+from scrapers.stumptown import scrape_stumptown
```

Comment:
might be better to just import scrapers and just do `scrapers.stumptown.scrape_stumptown` and intelli